adam-core/src/main/scala/org/bdgenomics/adam/rdd/read/AlignmentRecordDataset.scala

```diff
@@ -190,20 +190,20 @@ object AlignmentRecordRDD extends Serializable {
   def apply(ds: Dataset[AlignmentRecordProduct],
```

Comment:
This function could use scaladoc, the equivalent  ` def apply(ds: Dataset[GenotypeProduct],` function in `GenotypeDataset.scala` does have scaladoc and notes `..., populated from a SQL Dataset.`